dev/import-beats-resources/cisco/docs/README.md

```diff
@@ -0,0 +1,29 @@
+# Cisco Integration
+
+This integration is for Cisco network device's logs. It includes the following
+filesets for receiving logs over syslog or read from a file:
```

Comment:
When talking about integrations, we shouldn't use words like "filesets" or "metricsets" as they don't exist anymore. It's safer to replace these terms with "datasets".